.travis.yml

```diff
@@ -0,0 +1,11 @@
+sudo: false
+language: ruby
+cache: bundler
+rvm:
+  - 2.3.6
+  - 2.4.3
+  - 2.5.0
+script:
+  - export RAILS_ENV=test
```

Comment:
Same comment as I made [on the id-mapper PR](https://github.com/mysociety/id-mapper/pull/1/files#r174732491), I think Travis recommends using an `env:` key. Sorry!